README.md

```diff
@@ -334,6 +338,39 @@ that may not have their own acess controls at the Nginx layer. To learn more
 about how to use it, an admin should peruse the [groups](#groups) section of
 the documentetion.
 
+## Management
+
+Managing an identity provider can be a complex task requiring many different
+types of users. 
+
+### Operators
+
+An operator is a type of user that can manage EyeDP itself, but cannot manage
+users or groups. A user becomes an operator when they're added to a group that
+has the operator flag enabled, which can be done in the admin UI by an
+administrator.
+
+### Managers
+
+An operator is a type of user that can manage users and groups, but cannot
```

Comment:
The word operator should be replaced with manager.

Reply:
+1